docs/RunningOnDeviceIOS.md

```diff
@@ -24,16 +24,10 @@ In Xcode, select your phone as build target and press "Build and run"
 
 You have built a great app using React Native, and you are now itching to release it in the App Store. The process is the same as any other native iOS app, with some additional considerations to take into account.
 
-### Disabling the developer menu
+### Using the offline bundle & Disabling the developer menu
```

Comment:
@skatpgusskat Sorry for the delay. Yes, I like the updated title you proposed. Can you update the PR with that? Then I think we can ship this.